sustc-api/src/main/java/io/sustc/service/UserService.java

```diff
@@ -27,19 +27,19 @@ public interface UserService {
      * Deletes a user.
      *
      * @param auth indicates the current user
-     * @param mid  the user to be deleted
+     * @param mid  the user's {@code mid} to be deleted
```

Comment:
```suggestion
     * @param mid  the user to be deleted
```

imo the origin statement is clear enough, we are deleting a user whose corresponding mid is passed in the params, instead of 'deleting the mid'